README.md

```diff
@@ -241,15 +241,61 @@ CKB> account list
   ledger_id: 0x69c46b6dd072a2693378ef4f5f35dcd82f826dc1fdcc891255db5870f54b06e6
 ```
 
-The `ledger_id` shown is the public key hash for the path m/44'/309', which is the root Nervos path. Remember the `ledger\_id` given above for the latest commands. It will be
-used for ```<ledger-id>``` later on.
+The `ledger_id` shown is the public key hash for the path m/44'/309', which is the root Nervos path. the `ledger\_id` will be
+used for ```<ledger-id>``` argument in the `account import` command as described below.
 
-### Get Public Key ###
+If you have already imported the Ledger account, then `account list` command will instead give the account details.
+This will be shown even if the device is not connected.
 
-Get the public key:
+``` sh
+CKB> account list
```

Comment:
This is currently showing the output of the operation after the user has already imported their ledger account. Before that it is

```
- "#": 0
  account_source: ledger hardware wallet
  ledger_id: 0x27fe5acb022cd7b8be0eb7009d42ff4600c597d28b6affefcab6f7396d1311c2
```

Reply:
So should I remove this?

where should I mention this output from the 'account list' command?

Reply:
also note https://github.com/obsidiansystems/ledger-app-nervos/pull/83/files#diff-04c6e90faac2675aa89e2176d2eec7d8R237-R241

Reply:
No, I think we should show an `account list` prompt here, but I think we should show what the response is before the user has imported an account, which is what I included in my first comment. I like that we then show `account list` later on, too, after the user has done an import

Reply:
I just viewed the file and what you currently have actually looks great. No further changes necessary!